src/wp-admin/includes/media.php

```diff
@@ -3363,7 +3363,11 @@ function attachment_submitbox_metadata() {
 	if ( isset( $meta['filesize'] ) ) {
```

Comment:
The reason why this key is checks for, is it is set for audio and video files.

Reply:
See 
https://github.com/WordPress/wordpress-develop/blob/5bc27786d1568f68bd2c7d4a2f8230c73fc18a28/src/wp-admin/includes/media.php#L3669-L3671

https://github.com/WordPress/wordpress-develop/blob/5bc27786d1568f68bd2c7d4a2f8230c73fc18a28/src/wp-admin/includes/media.php#L3561-L3563